src/lib/misc.cpp

```diff
@@ -234,6 +234,34 @@ rnp_clear_debug()
     debugc = 0;
 }
 
+/* -1 -- not initialized
+    0 -- logging is off
+    1 -- logging is on
+*/
+int8_t _rnp_log_switch =
+#ifdef NDEBUG
+  -1 // lazy-initialize later
+#else
+  1 // always on in debug build
+#endif
+  ;
+
+void
+set_rnp_log_switch(int8_t value)
+{
+    _rnp_log_switch = value;
+}
+
+bool
+rnp_log_switch()
+{
+    if (_rnp_log_switch < 0) {
+        const char *var = getenv(RNP_LOG_CONSOLE);
+        _rnp_log_switch = !!var;
```

Comment:
I think it would be good to support `RNP_LOG_CONSOLE=0` instead of just checking for the presence of the env var. Sometimes it's easier to override an env var than it is to unset it.

Reply:
Agree! Let's update this and merge!